src/pages/MetricDetail.svelte

```diff
@@ -72,9 +72,12 @@
       <td>Relevant Bugs</td>
       <td>
         {#each metric.bugs as bug, i}
-          {#if bug.indexOf('http') > -1}
-            <a href={bug} title={bug} target="_blank"> {i + 1} </a>
-          {:else}<span>{bug}</span>{/if}
+          <a
+            href={typeof bug !== 'number' ? bug : `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug}`}
+            title={bug}
+            target="_blank">
+            {i + 1}
+          </a>
```

Comment:
While you're here, can you use the style @mhmohona used in #132? It's a little more descriptive to show the full URL.

(longer term we can come up with a better format for the bugs, but let's leave to later)